kmsconfig/config.go

```diff
@@ -82,11 +83,14 @@ func (c *Config) Load() error {
 }
 
 func (c *Config) LoadAndPopulate(config interface{}) error {
+	if os.Getenv("K8S") == "true" {
```

Comment:
Just a note - this isn't exclusively for k8s-compatible deployments, rather that we want to be _able_ to specify any config variable at runtime via an environment variable. While this has obvious uses for k8s, it's not the entire use case.

Reply:
Yeah, I'll rename it